web_form_sticky_header/static/src/css/sticky_header.css

```diff
@@ -0,0 +1,8 @@
+.oe_form header {
```

Comment:
The original specifies it must be within an openerp class. Should that not be also done for this one?

Reply:
https://github.com/OCA/OCB/blob/8.0/addons/web/static/src/css/base.css#L1922

I believe ronald is right here

change with .openerp .oe_form header {